doc/index.md

```diff
@@ -57,8 +55,19 @@ The detailed algorithm of spglib is described the following text:
 - Conda: <https://anaconda.org/conda-forge/spglib>
 - PyPI: <https://pypi.org/project/spglib/>
 
+## Contributors
+
+- Atsushi Togo (National Institute for Materials Science)
+- @lan496
+- @LecrisUT
+- [Even more people](https://github.com/spglib/spglib/graphs/contributors)
+
 ## Acknowledgments
 
+Spglib development is supported by
+
+- National Institute for Materials Science
```

Comment:
Specific grant or as part of your current position?

Reply:
I think both and may be something more if anything useful for contributors to keep contributing. This was to small start for my requirement. Maybe acknowledgement can have a dedicated page in the future 
 like https://www.aiida.net/sections/acknowledgements.html.

Reply:
Indeed, I think just moving that section to below the individual's acknowledgments would be good.

If you can provide the logo and link, I can suggest some formatting changes to make it look better.

>> Specific grant or as part of your current position?
>
> I think both and may be something more if anything useful for contributors to keep contributing.

I meant to say, do you have a specific grant to put there, or is it supported in perpetuity as part of your current position. The current wording would indicate the latter, but I want to confirm just in case. I think it would be good to be a bit more specific about that aspect.

Reply:
I see. For me currently, this information is enough.